src/javascript/functions/bookChallenge.js

```diff
@@ -58,7 +55,8 @@ export default function bookChallenge(data) {
              `;
 
             // GET AVAILABLE TIMES FROM BOOKING API
-            fetch("https://lernia-sjj-assignments.vercel.app/api/booking/available-times" + "?date=" + DateInput.value)
+            fetch("https://lernia-sjj-assignments.vercel.app/api/booking/available-times" +
+              "?date=" + DateInput.value + "&id=" + bookBtn[i].closest('.card').id)
```

Comment:
Let's hope Richards API will respond to this later on. LGTM !

Reply:
LGTY!